docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2"
+services:
+  webserver:
+    build: ./
+    environment:
+      MOODLE_DOCKER_DBTYPE: mariadb
+      MOODLE_DOCKER_DBNAME: moodle
+      MOODLE_DOCKER_DBUSER: moodle
+      MOODLE_DOCKER_DBPASS: "m@0dl3ing"
```

Comment:
Also Uh.....

Reply:
I stole it from https://github.com/moodlehq/moodle-docker/blob/master/base.yml#L14 also public, and also hopefully not used by anyone for real stuff.